theseus/src/api/pack/install_from.rs

```diff
@@ -66,12 +66,21 @@ pub enum EnvType {
 }
 
 #[derive(Serialize, Deserialize, Clone, Copy, Hash, PartialEq, Eq, Debug)]
-#[serde(rename_all = "kebab-case")]
```

Comment:
I got rid of this in the hopes that the others being explicit will encourage future devs to add a `#[serde(rename)]` and give it the correct explicit name.